2017-01-30-openwhisk-integration-with-serverless

```diff
@@ -0,0 +1,44 @@
+---
+title: Introducing OpenWhisk Integration with the Serverless Framework
```

Comment:
It seems that IBM now market it as "Apache OpenWhisk". Not sure if we should pick this up in our blog posts.